record-and-playback/core/systemd/bbb-rap-resque-worker.service

```diff
@@ -5,8 +5,8 @@ After=redis.service
 
 [Service]
 Type=simple
-ExecStart=/bin/sh -c '/usr/bin/rake -f ../Rakefile resque:workers >> /var/log/bigbluebutton/bbb-rap-worker.log'
-WorkingDirectory=/usr/local/bigbluebutton/core/scripts
+ExecStart=bundle exec rake -f Rakefile resque:workers
```

Comment:
This logs the output to stdout (i.e. the journal). I would like to change that elsewhere as well. If you feel this is too much of a change, we can reintroduce the `/bin/sh` hack.